Makefile

```diff
@@ -3,15 +3,15 @@ ci: test
 	npm run docs
 
 test: lint
-	npm run test
+	npm run $@
 
 lint: node_modules
-	npm run lint
+	npm run $@
 
 clean:
 	rm -rf node_modules
 
 node_modules: package.json
 	npm install
 
-.PHONY: node_modules
+.PHONY: clean
```

Comment:
> Can you explain why 'clean' is the preferable .PHONY target here?

Just so that there is not a file named clean which would mess things up.



> Using node_modules here means that npm install will run even if there is already a node_modules directory. Which is what you want in the event that package.json has been changed.

I don't think that is true, I believe that a PHONY target will ignore any prerequisites (in this case package.json) and always run `npm install` instead of only running it when package.json has been changed